install_script.sh.template

```diff
@@ -590,6 +590,38 @@ function install_installer() {
     return "$exit_status"
 }
 
+function remove_existing_packages_for_fips_flavor() {
+    local sudo_cmd="$1"
+    local os="$2"
+
+    local exit_status=0
+    # find if fips-proxy or existing agent is installed
+    if [ "$os" == "Debian" ]; then
+        $sudo_cmd apt list --installed "datadog-agent" "datadog-fips-proxy" 2>/dev/null | grep -q "datadog" || exit_status=$?
+    elif [ "$os" == "Red Hat" ]; then
+        $sudo_cmd yum list installed "datadog-agent" "datadog-fips-proxy" || exit_status=$?
+    elif [ "$os" == "SUSE" ]; then
+        $sudo_cmd zypper search -i "datadog-agent" "datadog-fips-proxy" || exit_status=$?
+    else
+        return 0
+    fi
+
+    if [ "$exit_status" -ne 0 ]; then
+        return 0
+    fi
+
+    echo -e "\033[34m\n* Removing the datadog-fips-proxy and existing datadog-agent packages\n\033[0m"
+    if [ "$os" == "Debian" ]; then
+        $sudo_cmd apt-get remove -y --force-yes "datadog-fips-proxy" "datadog-agent" || return $?
```

Comment:
will an error message surface if it returns a nonzero exit status?

Reply:
Hmm we're not redirecting stderr with these commands so it should be visible in the script? I can add out own error message here, we probably would not want to silently continue on a nonzero exit code (tbh I don't know what could cause a `apt-get|yum remove -y` could fail with a nonzero)

Reply:
added in https://github.com/DataDog/agent-linux-install-script/pull/327/commits/ae3c26717fc1bbfe80125409fa149455e66adc66

Reply:
thanks!